datafusion/sql/tests/sql_integration.rs

```diff
@@ -2962,36 +2962,37 @@ fn cte_with_no_column_names() {
 
 #[test]
 fn cte_with_column_names() {
-    let sql = "WITH \
-        numbers(a, b, c) AS ( \
-            SELECT 1, 2, 3 \
-        ) \
-        SELECT * FROM numbers;";
+    let sql = r#"WITH
+        numbers(a, b, c) AS (
+            SELECT 1, 2, 3
+        )
+        SELECT * FROM numbers;"#;
 
-    let expected = "Projection: numbers.a, numbers.b, numbers.c\
-        \n  SubqueryAlias: numbers\
-        \n    Projection: Int64(1) AS a, Int64(2) AS b, Int64(3) AS c\
-        \n      Projection: Int64(1), Int64(2), Int64(3)\
-        \n        EmptyRelation";
+    let expected = r#"Projection: numbers.a, numbers.b, numbers.c
+  SubqueryAlias: numbers
+    Projection: Int64(1) AS a, Int64(2) AS b, Int64(3) AS c
+      Projection: Int64(1), Int64(2), Int64(3)
+        EmptyRelation"#;
 
-    quick_test(sql, expected)
+    quick_test(sql, expected);
 }
 
 #[test]
 fn cte_with_column_aliases_precedence() {
```

Comment:
are the changes in this test related to unnest? Maybe we could break them out into their own PR for review

Reply:
This change is not related to unnest directly, but they are affected by adding the `UnnestExpression` layer, so they are better changed together in this PR.